test/wlog.lean

```diff
@@ -1,131 +1,75 @@
 /-
 Copyright (c) 2018 Simon Hudon. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Simon Hudon
+Authors: Simon Hudon, Johan Commelin
 -/
 import tactic.wlog
 
 section wlog
 
-example {x y : ℕ} (a : x = 1) : true :=
+example {x y : ℕ} : true :=
 begin
   suffices : false, trivial,
```

Comment:
These examples are pretty wacky, I have to say. :-) Why not just have the conclusion be `false` in the first place?

More importantly, perhaps we could have the tests include one or two easy but realistic examples, essentially as part of the documentation.

Reply:
Because you will get a warning "declaration uses sorry" if you don't use the suffices trick.

Reply:
I removed the true-false dance.